### PR TITLE
Disabling the reclustering in the MET correction and uncertainty tool

### PR DESCRIPTION
**Pull request by mmarionncern**
Following the discussion of https://indico.cern.ch/event/447792/?filterActive=1&showDate=all&showSession=4
this PR removes the jet clustering and enable the repro74X option in the shortcut functions used for MET reprocessing on top of miniAOD.

New features :
- MET extractor : allows to retrieve the raw MET from an existing slimmedMET collection; the recomputation of the raw MET is thus not needed anymore if not explicitely specified in the tool
- jet reclustering disabled per default,  existing slimmedJets are directly recalibrated using the last JECs and the Type1MET is defined accordingly
- the repro74X option has been added in the runMetCorAndUncFromMiniAOD function, allowing users to recompute the MET on top of 74X, X<12 miniAODs on the fly in a simplified way.
- the caloMET is now copied in the new slimmedMET collection
- automatic protection has been added against the change of process name used to create the first slimmed MET collection. Users won't have to switch manually the input tag to  slimmedMET__RECO anymore

Changes expected :
- gain of CPU performances
- physics untouched, rounding of PFcandidate quantities removed from the computations.

Has to be done on top of #11482

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by gpetruc** [comment]
Hi @mmarionncern,

This PR affects only the re-computation of type1 MET from miniAOD, not what's run in prooduction, ok?

Giovanni

**Comment by mmarionncern** [comment]
@gpetruc Yes, the type1MET correction is only computed at the miniAOD level so far.

File: PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py [hunk 1]
```diff
@@ -1165,7 +1173,39 @@ def recomputeRawMetFromPfcs(self, process, pfCandCollection, onMiniAOD, patMetMo
...
+            patMetModuleSequence += getattr(process, "met"+correctionLevel+postfix)
+
+
+    def updateJECs(self,process,jetCollection, patMetModuleSequence, postfix):
+        from PhysicsTools.PatAlgos.producersLayer1.jetUpdater_cff import patJetCorrFactorsUpdated
+        patJetCorrFactorsReapplyJEC = patJetCorrFactorsUpdated.clone(
+            src = jetCollection,
+            levels = ['L1FastJet', 
+                      'L2Relative', 
+                      'L3Absolute'],
```
Comment (gpetruc) [inline PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py]: Naive question: shouldn't there be also L2L3Residual?

Comment (mmarionncern) [inline PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py]: There is a specific case for data L1252 of the new version

**Pull request merged**